api/src/main/java/io/onetable/model/stat/PartitionValue.java

```diff
@@ -0,0 +1,32 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+ 
+package io.onetable.model.stat;
+
+import lombok.Builder;
+import lombok.Value;
+
+import io.onetable.model.schema.OnePartitionField;
+
+/** Associates the partition value with the partition field. */
+@Value
+@Builder
+public class PartitionValue {
```

Comment:
Wrapper for associating the partition field and range so we can have an ordered list of partition values instead of a map